lib/EarlyFunctionAnalysis/FunctionSummaryOracle.cpp

```diff
@@ -165,18 +165,24 @@ void importModel(Module &M,
     for (const model::CallSitePrototype &CallSite :
          Function.CallSitePrototypes()) {
 
+      AttributesSet Attributes;
+      for (auto &ToCopy : CallSite.Attributes())
+        Attributes.insert(ToCopy);
```

Comment:
not sure i understand the question. The model contains TrackingVectors, and the other datastructures unrelated to the model contain SortedVectors, so they can't be trivially copied. We can add a constructor to sortedvector and mutable set if you prefer.

Reply:
My question was why do we have to do this, it wasn't obvious to me that it was due to model containers having become tracking.

STL data structures have a constructor taking `.begin()` and `.end()`, what do you think?

Reply:
this ends up in the discussion about mutable set being unclear about which instance of values that compare equal ends up picking. It is the second issue from the top in this MR.

Reply:
The destination is being written so the tracking is entirely irrelevant.
Best thing would not to copy any form of tracking.

Reply:
it did copied before too, generated classes cannot have references. what changed is not the contained stuff, just that one field is a sorted vector and the source was a tracking vector. 

anyway this is unrelated to the issue of begin() end() constructor. I had that constructor in one of the earlier version of this pr and then it got removed because  the semantic was unclear when the input range had multiple copies of the same element.